extensions/tabs/scripts/matrix.js

```diff
@@ -0,0 +1,34 @@
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+
+// eslint-disable-next-line no-unused-vars
+class Matrix {
+  constructor () {
+    this.p = 0
```

Comment:
Looks like you can remove `this.p` and `this.n`, maybe that will allow you to remove `eslint`.